core/engine/src/module/mod.rs

```diff
@@ -223,6 +223,19 @@ impl Module {
         )
     }
 
+    /// Create a module that exports a single JSON value as the default export, from its
+    /// JSON string. This required the `json` feature to be enabled.
+    ///
+    /// # Errors
+    /// This will return an error if the JSON string is invalid or cannot be converted.
+    pub fn from_json_as_default(json: &str, context: &mut Context) -> JsResult<Self> {
+        let json_value = serde_json::from_str::<serde_json::Value>(json).map_err(|e| {
+            JsError::from_opaque(js_string!(format!("Failed to parse JSON: {}", e)).into())
+        })?;
+        let value: JsValue = JsValue::from_json(&json_value, context)?;
+        Ok(Self::from_value_as_default(value, context))
+    }
```

Comment:
This should use `builtins::Json::parse` instead of `serde_json`, to avoid coupling the engine with serde.

Also, `Module::parse_json` is fine. The specification declares that the function should be called `ParseJSONModule`, which assumes that the output will be on the default export, but we remove the suffix for simplicity.

Reply:
Thanks I was looking for that builtins.

Reply:
`parse_json` implies a return value of `Json` that is parsed. I did the change but I figure `json_module` would be better.

Reply:
Not really? JSON is just the name of the syntax used to generate a `JsObject`, it doesn't really make sense to return a "Json" value; in serde-json, for example, the `serde_json::from_str` doesn't return a `Json` type, but a `Value` type.